autoinstall/README.md

```diff
@@ -0,0 +1,17 @@
+# Autoinstall
+By using autoinstall we could automate the subiquity installer like what `preseed` is used for `debian-installer`, a.k.a. `d-i`.
+
+## Usage
+1. Upload your `user-data` to a www server. You may use `basic.yaml` as an example. Rename `basic.yaml` to be `user-data` and put it into a www server to serve it.
```

Comment:
Mind noting that there's a pre-baked password within that they may want to change?

Reply:
+1. Will do in the upcoming PR update.